spyder/plugins/ipythonconsole/plugin.py

```diff
@@ -901,6 +901,9 @@ def register_client(self, client, give_focus=True):
             if (self.main.projects is not None and
                     self.main.projects.get_active_project() is not None):
                 cwd_path = self.main.projects.get_active_project_path()
+        elif CONF.get('workingdir', 'startup/use_fixed_directory'):
+            cwd_path = CONF.get('workingdir', 'startup/fixed_directory',
+                                default=get_home_dir())

```

Comment:
Again, is this really necessary? I wouldn't like to add code here that's only needed by our tests.

Reply:
That is the change that makes the console start in the `'startup/use_fixed_directory'`, before it wasn't connected and that's why the bug was present

Reply:
Ok, no problem.